Sources/Danger/Danger.swift

```diff
@@ -7,16 +7,20 @@ import Foundation
 #endif
 import Logger
 
+let runner = DangerRunner()
+
+public var danger: DangerDSL {
+    return runner.dsl
+}
+
 // MARK: - DangerRunner
 
 final class DangerRunner {
-    static let shared = DangerRunner()
-
     let logger: Logger
     let dsl: DangerDSL
     var results = DangerResults()
 
-    private init() {
+    fileprivate init() {
         let isVerbose = CommandLine.arguments.contains("--verbose") || (ProcessInfo.processInfo.environment["DEBUG"] != nil)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/Danger/Danger.swift;
  Line: 24;
-->

- :warning: Line should be 120 characters or less: currently 124 characters (`line_length`)